src/utils/pinHelpers.js

```diff
@@ -24,4 +23,4 @@ async function expirePIN(user) {
   user.expiration = date;
   await user.save();
```

Comment:
aren't you still saving  the user object?

Reply:
Good catch. There's not really a case where we wouldn't want to save immediately upon expiring PIN since the user is already in the DB, but should we change it for consistency sake? It might be better to just keep it consistent.

Reply:
I think it might be easier to have every save user command in the requests to keep it consistent.

Reply:
Yep, I agree. Just pushed up that change.